utils/python/CIME/build.py

```diff
@@ -237,6 +240,9 @@ def case_build(caseroot, case, sharedlib_only=False, model_only=False):
     os.environ["CISM_USE_TRILINOS"]    = stringify_bool(cism_use_trilinos)
     os.environ["MPASLI_USE_ALBANY"]    = stringify_bool(mpasli_use_albany)
 
+    if get_model() == "acme" and mach == "titan" and compiler == "pgiacc":
```

Comment:
I believe that this can go into config_machines.xml  - without any new create_newcase arguments...